collectors/ebpf.plugin/README.md

```diff
@@ -39,18 +39,93 @@ section for details.
 
 ## Charts
 
-The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with three sub-menus: **File**, **VFS**, and
-**Process**. All the charts in this section update every second. The collector stores the actual value inside of its
-process, but charts only show the difference between the values collected in the previous and current seconds.
+The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with two sub-menus: **Socket**, and
+**Process**. The plugin also creates charts on different menus, like System Overview, Memory, Disks, Filesystem, 
```

Comment:
should menu names be enclosed in `backticks` (``)? we do that for chart names below

Reply:
A good point, I thought the bold around would be enough, @hugovalente-pm can you clarify this for us?

Reply:
> System Overview, Memory, Disks, Filesystem, Mount Points and Applications

Is neither bold nor inline code (with backticks)

Reply:
was trying to find other examples and could only find this one https://learn.netdata.cloud/docs/agent/collectors/apps.plugin where bold is used for **Applications**, **Users** and **User Groups**, but this was the only I could find not sure if there are other examples.

imo bold works ok as long as we do on all, like the place @underhood called, but this is just personal opinion

Reply:
@underhood this is the unique change I did not bring, I will wait for your answer to proceed.

Reply:
@thiagoftsm up to you then, wont block on this

Reply:
Thank you! I will move forward with @hugovalente-pm 's suggestion, but we will have more PRs related to these group of charts, so we will have opportunity to change them again.